client/web/src/components/widgets/options/ToolOptions.vue

```diff
@@ -4,7 +4,7 @@
 			<!-- TODO: Use `<component :is="" v-bind="attributesObject"></component>` to avoid all the separate components with `v-if` -->
 			<IconButton
 				v-if="option.kind === 'IconButton'"
-				:action="() => (option.message && sendToolMessage(option.message), option.callback && option.callback())"
+				:action="() => (option.message && sendToolMessage(option.message), option.callback && option.callback(), !option.message && !option.callback && comingSoon())"
```

Comment:
Is this tuple just to allow multiple calls in the callback? I think this should be a component method, and each of the `&&` should be an `if` statement instead. Maybe something like:

```javascript
let notImplemented = true;
if (option.message) {
    sendToolMessage(option.message);
    notImplemented = false;
}

if (option.callback) {
    option.callback()
    notImplemented = false;
}

if (notImplemented) {
    comingSoon();
}
```